packages/compiler/src/processTaggedTemplateExpression.ts

```diff
@@ -0,0 +1,48 @@
+import { Node, SyntaxKind, TaggedTemplateExpression } from "ts-morph";
+import { sheet } from "@kuma-ui/sheet";
+
+const parseCssTemplate = (cssTemplateLiteral: Node) => {
+  if (Node.isNoSubstitutionTemplateLiteral(cssTemplateLiteral)) {
+    const cssString = cssTemplateLiteral.getLiteralText();
+    return cssString ? sheet.parseCSS(cssString) : undefined;
+  }
+  return undefined;
+};
+
+export const processTaggedTemplateExpression = (
+  node: TaggedTemplateExpression,
+  bindings: Record<string, string>
+) => {
+  const tag = node.getTag();
+  // css``
+  if (Node.isIdentifier(tag) && tag.getText() === bindings["css"]) {
+    const cssTemplateLiteral = node.getTemplate();
+    const className = parseCssTemplate(cssTemplateLiteral);
+    if (className) {
+      node.replaceWithText(JSON.stringify(className));
+    }
+  }
+  // styled("xxx")``
+  else if (
+    Node.isCallExpression(tag) &&
+    tag.getExpressionIfKind(SyntaxKind.Identifier)?.getText() ===
+      bindings["styled"]
+  ) {
+    const cssTemplateLiteral = node.getTemplate();
+    const className = parseCssTemplate(cssTemplateLiteral);
```

Comment:
This function seems to be used with two APIs, `styled` and `css`, so the naming of `parseCssTemplate` feels a little off.

Reply:
renamed to `extractClassName`: https://github.com/kuma-ui/kuma-ui/pull/248/commits/ae0ffa6ae4338ceedf0c50b4f6ff9fdba42f797b